pkg/service/one2onerestore/model.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2025 ScyllaDB
+
+package one2onerestore
+
+import (
+	"github.com/pkg/errors"
+	"github.com/scylladb/scylla-manager/v3/pkg/scyllaclient"
+	. "github.com/scylladb/scylla-manager/v3/pkg/service/backup/backupspec"
+)
+
+// Target specifies what data should be restored and from which locations.
+type Target struct {
+	Location        []Location    `json:"location"`
+	Keyspace        []string      `json:"keyspace,omitempty"`
+	SourceClusterID string        `json:"source_cluster_id"`
+	SnapshotTag     string        `json:"snapshot_tag"`
+	NodesMapping    []nodeMapping `json:"nodes_mapping"`
+	BatchSize       int           `json:"batch_size,omitempty"`
+	Parallel        int           `json:"parallel,omitempty"`
+	Transfers       int           `json:"transfers"`
+	RateLimit       []DCLimit     `json:"rate_limit,omitempty"`
+	AllowCompaction bool          `json:"allow_compaction,omitempty"`
```

Comment:
We don;t need this parameter.
Compaction should be disabled to not waste CPU cycles and disk IO.
But it doesn't have an impact on the correctness of the restore.